dbms/src/Interpreters/ExpressionActions.cpp

```diff
@@ -1188,8 +1188,9 @@ bool ExpressionActions::checkColumnIsAlwaysFalse(const String & column_name) con
     /// Check has column in (empty set).
     String set_to_check;
 
-    for (auto & action : actions)
+    for (auto it = actions.rbegin(); it != actions.rend(); ++it)
```

Comment:
Why we changed the order?

Reply:
Cause we only need the last suitable action. It's a small improvement.

Reply:
I've also added `break` below.